### ISSUE TITLE
The shortcut to go to the message list does not work

### ISSUE THREAD

**Issue by stefanbanita** [juliano-lopes/accessibility-by-force#5]
For several days now, in the script WhatsappWebWithMoreAccessibility you cannot use the ALT + M shortcut to go to the message list.
The shortcuts to go to the chat list or to the edit box work well.
